src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const initialDateFromat = date.split(fromFormat[3]);
+  const result = [];
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    for (let j = 0; j < fromFormat.length - 1; j++) {
```

Comment:
Loop in the loop. O(n^2) complexity. 
Do you have any ideas on how to make it less complex?

Reply:
implemented without nested loops